src/System.Runtime.Intrinsics.X86/dir.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <AssemblyKey>MSFT</AssemblyKey>
```

Comment:
@weshaggard Should we use Open key instead? or is it ok to use MSFT

Reply:
Yes for any new assemblies they should use the Open key.